lib/View/mycourses_screen.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:flutter/material.dart';
+import 'package:gocast_mobile/View/utils/custom_bottom_nav_bar.dart';
+import 'settings_screen.dart';
+import 'package:gocast_mobile/View/utils/course_card_view.dart';
+class MyCourses extends StatelessWidget {
+  const MyCourses({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.white, // Replace with the exact color
+        title: const Text(
+          'GoCast',
+          style: TextStyle(color: Colors.black),
+        ), // Replace with the exact color
+        actions: [
+          IconButton(
+            icon: const Icon(
+              Icons.settings,
+              color: Colors.black,
+            ), // Replace with the exact color
+            onPressed: () {
+              // Settings action
+              Navigator.push(
+                context,
+                MaterialPageRoute(
+                  builder: (context) => const SettingsScreen(),
+                ),
+              );
+            },
+          ),
+        ],
+      ),
+      body: SingleChildScrollView(
```

Comment:
Refactor to use single component (e.g., or `CourseGrid`) that you then use in `mycourses_screen` and `publiccourses_screen` to avoid duplicate code.

Reply:
done